src/library/create_netconf_manager.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+import vspk.v5_0 as VSPK
```

Comment:
Should this be 5 or 6?

Reply:
Added support for v5 or v6 based on version